packages/circuit-ui/components/Table/components/SortArrow/SortArrow.tsx

```diff
@@ -14,15 +14,20 @@
  */
 
 /** @jsx jsx */
-import PropTypes from 'prop-types';
-import styled from '@emotion/styled';
+import { ButtonHTMLAttributes } from 'react';
```

Comment:
It gave me an error when I tried `HTMLProps`, so something must be different.

I found an explanation [here](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/36505#issuecomment-549394273) that says that `ButtonHTMLAttributes` basically narrows down the type of `type` to be only those supported by a button (`"button" | "reset" | "submit" | undefined`) instead of `string`

Reply:
Interesting, thanks for that link! I'll look into this more and might adopt this for other buttons in Circuit UI 💡